Source/inv.cpp

```diff
@@ -637,32 +730,6 @@ BOOL GoldAutoPlace(int pnum)
 	return done;
 }
 
-BOOL WeaponAutoPlace(int pnum)

```

Comment:
Did you check if the affected functions are bin exact for hellfire?

Reply:
Honestly didn't... I feel like it _will_ end up affecting Hellfire, since the Bard class allows dual wielding... let me check and update here.

Reply:
There is a bunch of Hellfire customizations in there as I suspected:

https://github.com/diasurgical/devilution/blob/81d57286991127c1f41a8da10d2d1e744cce7ab1/Source/inv.cpp#L989-L1002

What do you suggest in this case? Wait for Hellfire?

Reply:
Hmm, the changes aren't too bad. If I'm reading it right they made the Monk never auto equip weapons, and bard and barbarian only if it's a mace or sword (implemented in two completely different ways...).
I don't think that will be hard to implement in the change you have made here. And the conflict looks to be reasonable to deal with.

Reply:
Changes have now been translated into the new functions.